### DISCUSSION TITLE
primefaces theme do not change

### DISCUSSION THREAD

**Discussion by ashishchauhan99** [joinfaces/joinfaces#1242] [Q&A] [answered]
I am using the version 4.5.4 from the join faces with prime faces 10. In order to change the themes I added following dependency:
        <dependency>
            <groupId>org.primefaces.themes</groupId>
            <artifactId>all-themes</artifactId>
            <version>1.0.10</version>
        </dependency>
and added following property into spring-boot application.properties:
jsf.primefaces.theme=rocket

 However it does not seem to change anything. I build, restart but the theme always seem to be the same. 
 Could somebody help me here what wrong I am doing or is it not supported at all. 
 
 I have created a small hello world project which can be found here:
 https://github.com/ashishchauhan99/primestarter

**Comment by ashishchauhan99** [comment, marked as answer]
I was using the wrong property, the right property. The right was is:
joinfaces.primefaces.theme=rocket

**Reply by melloware** [reply]
Nice work!
